cluster-autoscaler/go.sum

```diff
@@ -191,6 +191,8 @@ github.com/go-openapi/swag v0.19.5/go.mod h1:POnQmlKehdgb5mhVOsnJFsivZCEZ/vjK9gh
 github.com/go-openapi/validate v0.19.5/go.mod h1:8DJv2CVJQ6kGNpFW6eV9N3JviE1C85nY1c2z52x1Gk4=
 github.com/go-ozzo/ozzo-validation v3.5.0+incompatible h1:sUy/in/P6askYr16XJgTKq/0SZhiWsdg4WZGaLsGQkM=
 github.com/go-ozzo/ozzo-validation v3.5.0+incompatible/go.mod h1:gsEKFIVnabGBt6mXmxK0MoFy+cZoTJY6mu5Ll3LVLBU=
+github.com/go-resty/resty/v2 v2.1.1-0.20191201195748-d7b97669fe48 h1:JVrqSeQfdhYRFk24TvhTZWU0q8lfCojxZQFi3Ou7+uY=
```

Comment:
The packngo client had external depencies that were not already vetted, so the Packet provider implements a minimal REST client in autoscaler.  I would expect something similar in this case.  gopkg.in/resty.v1 is already in go.mod, but for Linode v2 would be needed.

Reply:
I reviewed the PR for the Packet provider and now I know that external dependencies are not allowed anymore, I missed that information at first. I only use 3 API endpoints so I hope to substitute the linodego SDK with an internal implementation, I will work on it and update this PR.

Reply:
I've updated the PR implementing a minimal rest client.